hack/test-end-to-end.sh

```diff
@@ -225,12 +225,12 @@ echo "Log in as 'e2e-user' to see the 'test' project."
 
 # install the router
 echo "[INFO] Installing the router"
-openshift ex router --create --credentials="${KUBECONFIG}" --images="${USE_IMAGES}"
+openshift ex router --create --credentials="${CERT_DIR}/openshift-router/.kubeconfig" --images="${USE_IMAGES}"
```

Comment:
since we're generating certs as non-sudo, can we remove the `chmod` calls on "${CERT_DIR}/.../" above?